grid.v3/client.go

```diff
@@ -300,16 +299,18 @@ func (c *Client) getWireClient(ctx context.Context, nsReceiver string) (WireClie
 	return cc.client, nil
 }
 
```

Comment:
@epsniff Soooooorry, yes that's been there forever in SQL, totally didn't think about that... changed it back to minus!